### PR TITLE
Docs/des.md

### PR DESCRIPTION
**Pull request by hwamzzx**
# What does this PR do?
DES에 대한 설명을 간략히 작성하였습니다.
DES 자체에 대한 소개와 어떤 방식으로 암호화가 이루어지는 지에 대해 넣었습니다.
잘못된 부분 혹은 좀 더 추가가 필요한 부분 피드백 주시면 감사하겠습니다.

Home > content > Basic Cryptography > DES.md 페이지를 만들었습니다.

## Before reviewing
- [x] Check for missing / redundant content
- [x] Grammar Check
- [x] Check if the content is accurate

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: content/Basic Cryptography/DES.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+2. **Swapper Stage**: In the swapper stage, the value computed from the XOR operation is placed on the right, while the original 32-bit right half is moved to the left. This can be expressed as: $$ L_i = R_{i-1} $$
+Thus, the i-th round can be represented as follows.
+$$ L_i = R_{i-1} $$
+$$ R_i = L_{i-1} \oplus F(R_{i-1}, k_i) $$
+
+#### F function
+##### Expansion P-Box
+The **Expansion P-Box** is a process that permutes the order of the bits of a 32-bit input value while expanding the output to 48 bits.
+The method for expanding the input value is as follows:
+![[Pasted image 20241103212213.png]]
```
Comment (healthykim) [inline content/Basic Cryptography/DES.md]: Change the image name to "images/DES(1)" (This is the convention in this github)

File: content/Basic Cryptography/DES.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+> DES encrypts 64-bit plaintext using a 56-bit key to produce a 64-bit ciphertext.
+
+## History
+In 1973, the National Bureau of Standards (NBS) in the United States recognized the need for a standardized encryption method and called for proposals for what would become the DES (Data Encryption Standard). In 1977, NBS selected a modified version of an encryption algorithm called Lucifer, developed by IBM, to become DES.
+The National Security Agency (NSA) made modifications to Lucifer, including reducing the key length from 128 bits to 56 bits and altering the structure of the S-Box. This led to speculation that the U.S. intelligence agencies could more easily decrypt messages encrypted with DES. Despite these suspicions, DES went on to become the most widely used block cipher worldwide for 30 years after its publication.
+When DES was first introduced in the 1970s, a 56-bit key length provided adequate security. However, with advancements in computing, DES is no longer considered secure. To address these limitations, multiple DES techniques were introduced, notably Triple DES (3DES), which applies DES three times and provides 112 bits of security, making it more robust.
```
Comment (healthykim) [inline content/Basic Cryptography/DES.md]: EFF's cracking hardware and distributed.net were also important events contributed to its deprecation.

File: content/Basic Cryptography/DES.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+In 2005, NIST officially retired simple DES. Triple DES was also deprecated in 2019, and its usage was prohibited by the end of 2023, except for processing already encrypted data. Today, DES has been replaced by the more secure and powerful [[AES]].
+
+## Design
+High-level description of DES is as following:
+
+1. Initial Permutation (IP)
+2. 16 Rounds of F function and Swap
+	* F Function
```
Comment (healthykim) [inline content/Basic Cryptography/DES.md]: Please mention the full name of F function (Feistel Function)

File: content/Basic Cryptography/DES.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+		* Expansion P-Box
+		* XOR with Round Key
+		* S-Box Substitution
+		* P-Box Permutation
+	* Swap
+1. Inverse Initial Permutation (IP⁻¹)
```
Comment (healthykim) [inline content/Basic Cryptography/DES.md]: Indexing error? (1->2->1->2)

**Review by healthykim — APPROVED** [review]
LGTM!

**Comment by hunjyeong** [comment]
I think the PET > TEE part is well written.
I think it would be good to add more content about working on blockchain projects after looking at the contents of the link.
[https://collective.flashbots.net/t/blockchain-projects-working-with-trusted-execution-environments/3539]

**Comment by yijun-lee** [comment]
@abell1689neo

**Pull request merged**